README.md

```diff
@@ -1,98 +1,169 @@
-
-
-**Screenshot**: If the software has visual components, place a screenshot after the description; e.g.,
-
-![](https://raw.githubusercontent.com/cfpb/open-source-project-template/master/screenshot.png)
-
-
-## Dependencies
-
-Describe any dependencies that must be installed for this software to work. 
-This includes programming languages, databases or other storage mechanisms, build tools, frameworks, and so forth.
-If specific versions of other software are required, or known not to work, call that out.
+* [Flask](http://flask.pocoo.org/)
+* [Python icalendar](http://icalendar.readthedocs.org/en/latest/) 
+* [Python Dateutil](https://dateutil.readthedocs.org/en/latest/)
+* [Python mock (for the tests)](http://www.voidspace.org.uk/python/mock/)
 
 ## Installation
 
-Detailed instructions on how to install, configure, and get the project running.
-This should be frequently tested to ensure reliability. Alternatively, a link to
-another page is fine, but it's important that this works.
-
-## Configuration
-
-If the software is configurable, describe it in detail, either here or in other documentation to which you link.
-
-## Usage
-
-Show users how to use the software. 
-Be specific. 
-Use appropriate formatting when showing code snippets.
-
-## How to test the software
-
-If the software includes automated tests, detail how to run those tests.
+To clone and install flask-eventics locally in an existing Python 
+[`virtualenv`](https://virtualenv.pypa.io/en/latest/):
 
-## Known issues
+```shell
+$ git clone https://github.com/cfpb/flask-eventics
+$ pip install -e flask-eventics
+```
 
-Document any known significant shortcomings with the software.
+Note: this installs flask-eventics in 'editable' mode.
 
-## Getting help
+flask-eventics can be installed directly from Github:
 
-Instruct users how to get help with this software; this might include links to an issue tracker, wiki, mailing list, etc.
+```shell
+$ pip install git+https://github.com/cfpb/flask-eventics
+```
 
-**Example**
+Once installed, the flask-eventics blueprint simply needs to be
+registered with your flask application:
 
-If you have questions, concerns, bug reports, etc, please file an issue in this repository's Issue Tracker.
+```python
+from flask import Flask
+from flask_eventics import eventics
 
-## Getting involved
+app = Flask(__name__)
+app.register_blueprint(eventics)
+```
 
-This section should detail why people should get involved and describe key areas you are
-currently focusing on; e.g., trying to get feedback on features, fixing certain bugs, building
-important pieces, etc.
+If using [Sheer](https://github.com/cfpb/sheer), flask-eventics can be
+added to your site's `_settings/blueprints.json` file.
 
-General instructions on _how_ to contribute should be stated with a link to [CONTRIBUTING](CONTRIBUTING.md).
+```json
+{
+  "flask_eventics": {
+    "package": "flask_eventics",
```

Comment:
It could be either... I import `eventics` in the __init__.py file for convenience outside of our own conventions. I can change the readme easily enough though.